README.md

```diff
@@ -21,14 +21,13 @@ Then apply the `LazyImportsMixin` mixin (or `LazyImportsBehavior`) to your eleme
 ### Polymer 2.0 – LazyImportsMixin
 
 ```html
-<link rel="import" href="../../polymer/polymer-element.html">
-<link rel="import" href="../../paper-button/paper-button.html">
+<link rel="import" href="../../polymer/polymer.html">
```

Comment:
@abdonrd I think this got accidentally switched back to polymer instead of the element?